java/bundles/org.eclipse.set.application/OSGI-INF/l10n/bundle.properties

```
@@ -47,4 +47,9 @@ TableMenuService_Table=Zeige in Tabelle
 ControlAreaCombo_All_ControlArea=Alle Stellbereiche
 ControlAreaCombo_All_Objects_Value=Gesamter Planungsinhalt
 ControlAreaCombo_Default_Area_Name=Stellbereich %d
-ControlAreaCombo_PlanningArea_Value=Planungsbereich
\ No newline at end of file
+ControlAreaCombo_PlanningArea_Value=Planungsbereich
+
+IncompleteFileDialog_Title=Unvöllständige PlanPro-Datei
+IncompleteFileDialog_Message=Die geladende Datei ist für den Anwendungsfall PT 1 nicht vollständig. Es kann fehlerhaften oder unvollständigen Darstellungen kommen.
+InvalidFileDialog_Title=Ungültiges PlanPro-Modell
+InvalidFileDialog_Message=Die geladende Datei enthält kein valides Modell im PlanPro-Format. Es kann unvorhergesehenen Fehlern und Abstürzen kommen.
```

@TruongQuangSB Just some grammar and typo stuff

```suggestion
IncompleteFileDialog_Message=Die geladene Datei ist für den Anwendungsfall PT 1 nicht vollständig. Es kann zu fehlerhaften oder unvollständigen Darstellungen kommen.
InvalidFileDialog_Title=Ungültiges PlanPro-Modell
InvalidFileDialog_Message=Die geladene Datei enthält kein valides Modell im PlanPro-Format. Es kann zu unvorhergesehenen Fehlern und Abstürzen kommen.
```